pixel-saver@deadalnix.me/schemas/org.gnome.shell.extensions.pixel-saver.gschema.xml

```diff
@@ -1,9 +1,14 @@
 <schemalist>
+	<enum id="org.gnome.shell.extensions.pixel-saver.buttonPosition">
+		<value value="0" nick="Left" />
```

Comment:
Instead of 'left' and 'right' it could be better to use more descriptive names like 'after-menu' to make it easier to add more positions in the future if need be?